ui/qt/src/Ariadne/UI/Qt/Widgets/WalletInfo.hs

```diff
@@ -138,26 +137,13 @@ handleWalletInfoEvent UiLangFace{..} ev = do
   WalletInfo{..} <- ask
   lift $ case ev of
     WalletInfoSelectionChange selectionInfo -> do
-      QLabel.setText balanceLabel ("" :: String)
-      mCommandId <- atomicModifyIORef' balanceCommandId $ \cid -> (Nothing, cid)
-      whenJust (mCommandId >>= cmdTaskId) $ void . langPutUiCommand . UiKill
-      whenRightM (langPutUiCommand UiBalance) $ \cid -> do
-        QLabel.setText balanceLabel ("calculating..." :: String)
-        writeIORef balanceCommandId $ Just cid
-
-      whenJust selectionInfo $ liftIO . \case
-        UiSelectionWallet UiWalletInfo{..} -> whenJust uwiLabel $ QLabel.setText itemNameLabel . toString
-        UiSelectionAccount UiAccountInfo{..} -> whenJust uaciLabel $ QLabel.setText itemNameLabel . toString
-
-    WalletInfoBalanceCommandResult commandId result -> do
-      mCommandId <- readIORef balanceCommandId
-      when (mCommandId == Just commandId) $ do
-        writeIORef balanceCommandId Nothing
-        case result of
-          UiBalanceCommandSuccess balance -> do
-            QLabel.setText balanceLabel $ toString balance
-          UiBalanceCommandFailure err -> do
-            QLabel.setText balanceLabel $ toString err
+      let (itemName, balance) =
+            case selectionInfo of
```

Comment:
6 spaces?

Reply:
2, if you count from `(`. `stylish-haskell` :man_shrugging: 
Also, if you do 4 spaces from `let`:

```/.../ariadne/ui/qt/src/Ariadne/UI/Qt/Widgets/WalletInfo.hs:141:11: error:
    parse error (possibly incorrect indentation or mismatched brackets)
    |
141 |           case selectionInfo of```

The same with 2.

Reply:
Riiight, dat schizophrenic `let` indentation parsing.
I usually put a line feed after `let` in such cases